core/Cargo.toml

```diff
@@ -79,6 +80,7 @@ version = "0.4.42"
 default = []
 lzma = ["lzma-rs", "swf/lzma"]
 avm_debug = []
+tracy_avm = ["dep:tracy-client"]
```

Comment:
BTW How can this be enabled from a `cargo run` command? :thinking:

Reply:
Never mind, see above: `cargo run --features tracy,ruffle_core/tracy_avm`
I kept trying `core/tracy_avm`, and `ruffle_core/tracing_avm`...